src/Controller/Link/Delete.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Link;
+
+use App\Entity\Link;
+use App\Repository\LinkRepositoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+class Delete
+{
+    private LinkRepositoryInterface $repository;
+
+    public function __construct(LinkRepositoryInterface $repository)
+    {
+        $this->repository = $repository;
+    }
```

Comment:
То же самое, просто подключить rector и использовать последние возможности языка.
```suggestion
           public function __construct(private LinkRepositoryInterface $repository)
    {
    }
```